index.rst

```diff
@@ -2,28 +2,99 @@
+
+As shown in :sqr:`034` :cite:`SQR-034`, the EFD has of an `influxDB <https://www.influxdata.com/>`__ time series database, that stores the data produced by the system.
+The release of influx2.0, brings a number of important updates, amongst which two are relevant for stream processing; `flux`_ and `influxDB task`_.
+
+.. _flux: https://docs.influxdata.com/flux/v0.x/
+.. _influxDB task: https://docs.influxdata.com/influxdb/cloud/process-data/get-started/
+
+
+`flux`_ is a new functional data scripting language that is designed to process data streams.
+It contains extensive libraries that allows users to express complex operations, compatible with what can be done with programming languanges.
+Furthermore, `influxDB task`_ provides a way to schedule `flux`_ scripts.
+
+
+Each Catcher "rule" or "alarm" would be comprised of an `influxDB task`_ to execute a user defined `flux`_ script periodically.
+The `flux`_ script performs a set of actions/calculations based on EFD data, additional processes/tasks can be triggered depending on the results of these calculations and results can be stored in the EFD for future reference.
+
+Below we describe how some of the Catcher functionality would be implemented.
+
+Alarms
+------
+
+One of the key features expected from the Catcher was the ability to issue alarms that can be displayed in LOVE.
+One option to support this feature would be to create a custom LOVE producer that can be interacted with using a REST API.
+The alert information would be persisted in the EFD by the `flux`_ script, which would then post a message to the producer of the new alert.
+The producer would then read the alert informatino from the EFD and display it.
 
-The First-Look Analysis and Feedback Functionality (FAFF) working group, identified the need for a service that is capable of executing user-defined operations/computation following user-defined rules. This service has been referred  to as Catcher. In essence, the Catcher is not that different from the Watcher or other event handling off-the-shelf technologies like Kapacitor. This technote explores the idea of using influx2.0 event processing capabilities to as an alternative implementation for the Catcher.
+Process Image Data
+------------------
 
-Add content here
-================
+Another important feature of the Catcher was the capability to process camera image data using the OCPS.
+The OCPS is a CSC that is capable of executing DM-pipeline processes.
+It is going to be key in allowing remote processing of Main Camera data during commissioning and operations.
 
-Add content here.
-See the `reStructuredText Style Guide <https://developer.lsst.io/restructuredtext/style.html>`__ to learn how to create sections, links, images, tables, equations, and more.
+If the Catcher is not a CSC, it won't be able to interact direcly with OCPS to process image data.
+One alternative would be to interact with the OCPS remote backend direcly, instead of going through the CSC.
```

Comment:
Does the OCPS remote backend offer an HTTP-based API to trigger image processing? Perhaps explain in more detail the existing mechanism.